### PR TITLE
Clean up three early release announcements

### PR DESCRIPTION
**Pull request by sftim**
Mostly, this fixes the rendering for https://k8s.io/blog/2015/05/Kubernetes-Release-0170/

We typically don't revise old blog articles; I recommend making an exception because:
- these are release announcement articles
- we do want old articles to at least render correctly

### CODE REVIEW

**Review by dipesh-rawat — COMMENTED** [review]
Updating these old release announcement blog posts to render correctly makes sense.

/lgtm

[Preview Blog 1](https://deploy-preview-46733--kubernetes-io-main-staging.netlify.app/blog/2015/05/kubernetes-release-0170/)
[Preview Blog 2](https://deploy-preview-46733--kubernetes-io-main-staging.netlify.app/blog/2015/11/kubernetes-1-1-performance-upgrades-improved-tooling-and-a-growing-community/)
[Preview Blog 3](https://deploy-preview-46733--kubernetes-io-main-staging.netlify.app/blog/2015/04/kubernetes-release-0150/)

File: content/en/blog/_posts/2015-11-00-Kubernetes-1-1-Performance-Upgrades-Improved-Tooling-And-A-Growing-Community.md [hunk 1]
```diff
@@ -32,25 +32,22 @@ Some of the work we’d like to highlight includes:
...
 > “The tremendous customer demand we’re seeing to run containers at scale with Kubernetes is a critical element driving growth in our professional services business at Redapt. As a trusted advisor, it’s great to have a tool like Kubernetes in our tool belt to help our customers achieve their objectives.” – Paul Welch, SR VP Cloud Solutions, Redapt
 
->
-
-As we mentioned above, we would love your help:  
+As we mentioned above, we would love your help:
 
 - Get involved with the Kubernetes project on [GitHub](https://github.com/kubernetes/kubernetes)&nbsp;
 - Connect with the community on [Slack](http://slack.kubernetes.io/)
 - Follow us on Twitter [@Kubernetesio](https://twitter.com/kubernetesio) for latest updates&nbsp;
```
Comment (ricardoamaro) [inline content/en/blog/_posts/2015-11-00-Kubernetes-1-1-Performance-Upgrades-Improved-Tooling-And-A-Growing-Community.md:49]:
```suggestion
- Get involved with the Kubernetes project on [GitHub](https://github.com/kubernetes/kubernetes)
- Connect with the community on [Slack](http://slack.kubernetes.io/)
- Follow us on Twitter [@Kubernetesio](https://twitter.com/kubernetesio) for latest updates
```

Comment (ricardoamaro) [reply content/en/blog/_posts/2015-11-00-Kubernetes-1-1-Performance-Upgrades-Improved-Tooling-And-A-Growing-Community.md:49]: I spotted other "&nbsp;" in this file.

Comment (sftim) [reply content/en/blog/_posts/2015-11-00-Kubernetes-1-1-Performance-Upgrades-Improved-Tooling-And-A-Growing-Community.md:49]: I could clean those up, but it'd need a new LGTM. I'll leave this PR as a minimal cleanup.

**Review by divya-mohan0209 — COMMENTED** [review]
Thanks for these changes, @sftim !
/approve

**Pull request merged**